asv_bench/benchmarks/algos/isin.py

```diff
@@ -297,32 +300,47 @@ def setup(self, dtype, MaxNumber, series_type):
             array = np.arange(N) + MaxNumber
 
         self.series = Series(array).astype(dtype)
-        self.values = np.arange(MaxNumber).astype(dtype)
+
+        if is_extension_array_dtype(dtype):
+            vals_dtype = dtype.type
```

Comment:
should this be more specific to Int64Dtype/Float64Dtype?  we we had e.g. PeriodDtype then the astype on L308 would break

Reply:
Sounds good, have limited to those to make clearer what can pass through here in current form